cps-wallet-connector/README.md

```diff
@@ -0,0 +1,202 @@
+---
+CPS: ?
+Title: Cardano Wallet Connectors
+Status: Open
+Category: Wallets
+Authors:
+    - Ryan Williams <ryan.williams@iohk.io>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/619
+Created: 2023-07-28
+---
+
+## Abstract
+Wallets are a foundational element of web 3, being the primary interface between users and blockchains.
+Wallet-web bridges allow users to connect their wallet to web-based stacks, allowing a wide range of specialized experiences.
+
+Wallet web-bridge standards consist of two parts; connection standard and API.
+With the connection standard defining how the wallet and dApp initiate communication, this could be over injected Javascript object, WebRTC, WebTorrent, etc.
+The API defines what communications look like between the dApp and wallet post connection.
+
+This problem statement is concerned with the issues surrounding Cardano's current and future wallet-web bridges.
+These interfaces are difficult to define and historically have been even harder to iterate upon.
+We wish to provide a comprehensive catalog of the current offerings; and their drawbacks to be able to make suggestions on future standards.
+
+### Acknowledgments
+
+<details>
+  <summary><strong>First workshop - 2023-11-27</strong></summary>
+
+  I would like to thank those that contributed to the first workshop hosted by Adam Dean and Ryan Williams.
+  - George Flerovsky
+  - Beatrice Anihiri
+  - Denis Kalinin
+  - Evgenii Lisitskii
+  - George APEX Pool
+  - George Humphreys
+  - Hernán Rajchert
+  - Jack Rousian
+  - Joshua Marchand
+  - Ken Fritschy
+  - Ken-Erik Ølmheim
+  - Leo H
+  - Marcel Baumberg
+  - Martynas Kazlaukas
+  - Michael Chappell
+  - Mircea Hasegan
+  - Nicolas Ayotte
+  - Rhys Bartels-Wallet
```

Comment:
```suggestion
  - Rhys Bartels-Waller
```
_Bartels-Wallet_ would indeed be an appropriate surname, but... 😄 https://au.linkedin.com/in/rhyslbw

Reply:
That is a fun typo! 🤓